packages/playwright-client/types/types.d.ts

```diff
@@ -14918,10 +14918,7 @@ export interface BrowserType<Unused = {}> {
      */
     downloadsPath?: string;
 
-    /**
-     * Specify environment variables that will be visible to the browser. Defaults to `process.env`.
-     */
-    env?: { [key: string]: string|number|boolean; };
+    env?: { [key: string]: string|undefined; };
```

Comment:
this is breaking for no reason

Reply:
I'm aligning it with Node.ProcessEnv, looks like a bugfix to me we can afford.